dashboard/app/views/levels/_level_group.haml

```diff
@@ -0,0 +1,49 @@
+- app = 'level_group'
+- data = @level.properties
+:javascript
+  window.levelData = #{data.to_json};
+
+= render partial: 'levels/common_audio'
+
+- data["levels"].each_with_index do |level_name, index|
+  - level_blob = Level.find_by_name(level_name)
+  - @level = level_blob
+  - # Parse the array of the level resuts; see sample below in JavaScript.
+  - if @last_attempt
+    - last_attempt = JSON.parse(@last_attempt).each.select{|item| item["level_id"]==@level.id}.first["result"]
+  - else
+    - last_attempt = ""
+  = render partial: 'levels/multi', locals: {index: index, standalone: false, last_attempt: last_attempt }
+
+-# One submisison button for this level.
+= render partial: 'levels/dialog', locals: {app: app, data: data}
+
+
+:javascript
+
+  getResult = function()
```

Comment:
do you need var here?

In any case I think we are preferring to put any significant javascript code in .js files where lint would catch things like this